crates/extensions/tedge_log_manager/src/error.rs

```diff
@@ -0,0 +1,28 @@
+#[allow(clippy::enum_variant_names)]
+#[derive(thiserror::Error, Debug)]
+
+pub enum LogManagementError {
+    #[error(transparent)]
+    FromMqttError(#[from] tedge_mqtt_ext::MqttError),
+
+    #[error("Failed to parse response with: {0}")]
+    FromSerdeJsonError(#[from] serde_json::Error),
+
+    #[error(transparent)]
+    FromHttpError(#[from] tedge_http_ext::HttpError),
+
+    #[error(transparent)]
+    FromChannelError(#[from] tedge_actors::ChannelError),
```

Comment:
Okay, the issue is deeper and caused by `self.http_proxy.await_response(request).await?` mixing channel errors with HTTP errors. This will have to be improved by in a follow-up task.